extension/public/popup.js

```diff
@@ -0,0 +1,104 @@
+/********** Enabled Sites **********/ 
+
+// UI references
+const toggleList = document.getElementById("toggleList");
+const redditSwitch = document.getElementById("redditSwitch");
+const imgurSwitch = document.getElementById("imgurSwitch");
+const twitterSwitch = document.getElementById("twitterSwitch");
```

Comment:
Initially, I think we should have imgur/twitter permanently disabled - it will be easier if we target a single website before expanding to others. We should still show them as a "thing to come" imo. Thoughts?

Reply:
I'm open to that, given our time constraint. I'll raise a different issue for it once we get input from our other team members.

Reply:
Sure, seems reasonable to me.